packages/inter-protocol/src/auction/offerBook.js

```diff
@@ -25,12 +25,14 @@ const nextSequenceNumber = () => {
 
 /**
  * @typedef {{
- * seat: ZCFSeat,
- * wanted: Amount<'nat'>,
- * seqNum: NatValue,
- * received: Amount<'nat'>,
- * } &  {exitAfterBuy: boolean} & ({ bidScaling: Pattern, price: undefined } | { bidScaling: undefined, price: Ratio})
- * } BidderRecord
+ *   seat: ZCFSeat;
+ *   wanted: Amount<'nat'>;
+ *   seqNum: NatValue;
+ *   received: Amount<'nat'>;
+ * } & { exitAfterBuy: boolean } & (
+ *     | { bidScaling: Pattern; price: undefined }
+ *     | { bidScaling: undefined; price: Ratio }
```

Comment:
this answers my earlier question: yes, the tool leaves it alone if we write these as 1 line each instead of 3